www/_posts/2023/2023-11-03-chapters-video-request.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post.html
+title: "Chapters video request"
+tags: [PyLadies, Conference, Global]
+author: Denny Perez
+author_link: https://twitter.com/dennyperez18
+---
+
+Chapters video request
```

Comment:
Let's remove lines 9-12. We will use the subject as the blog post title. So the blog post will just start with "Hello amazing PyLadies!"